src/Microsoft.Azure.EventHubs/Microsoft.Azure.EventHubs.csproj

```diff
@@ -5,7 +5,7 @@
     <AssemblyTitle>Microsoft.Azure.EventHubs</AssemblyTitle>
     <VersionPrefix>1.0.3</VersionPrefix>
```

Comment:
><VersionPrefix>1.0.3</VersionPrefix> [](start = 4, length = 36)

considering this is a breaking change (raising the target framework), update the versionPrefix to 2.0

Reply:
Done